src/libs/actions/App.js

```diff
@@ -98,7 +97,6 @@ function getAppData(shouldSyncPolicyList = true) {
 
     // We should update the syncing indicator when personal details and reports are both done fetching.
     return Promise.all([
-        PersonalDetails.fetchPersonalDetails(),
```

Comment:
Ah from the Pattern B design doc, it seems like we won't display syncing status anymore, so that's not something that we need to handle anymore.

>Since we won’t use the avatar to show syncing/offline status, we can remove [this](https://github.com/Expensify/App/blob/6cd580794923e2a1985720effa646ff8e7506503/src/pages/settings/InitialSettingsPage.js#L161) usage and replace it with just [this code](https://github.com/Expensify/App/blob/fdf2fe46b3fbaf79bf170a2704f20fe0f35e83c6/src/components/AvatarWithIndicator.js#L100-L106).

Reply:
Nice.